### DISCUSSION TITLE
What are the placeholder for chat bot

### DISCUSSION THREAD

**Discussion by maplepy** [AntiCope/meteor-rejects#126] [bruh] [answered]
I know of `server.time` `server.tps` `player.pos `but what else have we access to?

**Comment by C10udburst** [comment, marked as answer]
Features like `.say`, Discord presence, ChatBot or InteractionMenu use Starscript. Here is a list of avaliable placeholders:
https://github.com/MeteorDevelopment/meteor-client/wiki/Starscript
